circuit_knitting/forging/entanglement_forging_ground_state_solver.py

```diff
@@ -143,6 +144,9 @@ def __init__(
             backend_names: Backend name or list of backend names to use during parallel computation
             options: Options or list of options to be applied to the backends
             mo_coeff: Coefficients for converting an input problem to MO basis
+            hf_energy: The Hartree-Fock energy to use at each iteration. If this is left unset,
+                the energy will be calculated with quantum experiments. See :ref:`Fixing the Hartree-Fock bitstring`
```

Comment:
```suggestion
                the energy will be calculated using quantum experiments. See :ref:`Fixing the Hartree-Fock bitstring`
```
This is a bit more clear to me.